src/server/lobby/LobbyDirector.cpp

```diff
@@ -413,20 +374,19 @@ void LoginDirector::HandleEnterRanch(
       LobbyCommandEnterRanchOK response{
         .ranchUid = user.ranchUid,
         .code = 0x44332211, // TODO: Generate and store in the ranch server instance
-        .ip = (uint32_t) addr.s_addr,
-        .port = port
-      };
+        .ip = (uint32_t)addr.s_addr,
```

Comment:
You have to send the advertised ranch server ip and port in here

Reply:
This won't resolve hostnames and will only work if the address in the settings is an IP address. It's fine for now i suppose